src/watch/mod.rs

```diff
@@ -102,10 +103,9 @@ pub fn step_callback(pcx: &mut PrirodaContext) {
                     }
                 }
 
-                alloc_trace.trace_points.push((
-                    *pcx.step_count,
-                    AllocTracePoint::Changed(alloc.clone()),
-                ));
+                alloc_trace
+                    .trace_points
+                    .push((*pcx.step_count, AllocTracePoint::Changed(alloc.clone())));
             } else if let Some(&(_, AllocTracePoint::Deallocated)) = alloc_trace.trace_points.last()
             {
```

Comment:
```suggestion
            } else if let Some(&(_, AllocTracePoint::Deallocated)) = alloc_trace.trace_points.last() {
```

Does rustfmt leave this intact?